data/mongodb/v0/config/tasks.json

```diff
@@ -0,0 +1,457 @@
+            "reference",
+            "template",
+            "value"
+          ],
+          "title": "Collection Name",
+          "type": "string"
+        },
+        "data": {
+          "description": "The data to be inserted",
+          "instillAcceptFormats": [
+            "semi-structured/*","structured/*","object"
+          ],
+          "instillShortDescription": "JSON Data",
+          "instillUIOrder": 2,
+          "instillUpstreamTypes": [
+            "reference",
+            "template",
+            "value"
+          ],
+          "title": "Data"
+        }
+      },
+      "required": [
+        "database-name",
+        "collection-name",
+        "data"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "instillUIOrder": 0,
+      "properties": {
+        "status": {
+          "description": "Insert status",
+          "instillFormat": "string",
+          "required": [],
+          "instillUIOrder": 0,
+          "title": "Status",
+          "type": "string"
+        }
+      },
+      "required": [
+        "status"
+      ],
+      "title": "Output",
+      "type": "object"
+    }
+  },
+  "TASK_FIND": {
+    "instillShortDescription": "Perform a find operation",
+    "input": {
+      "instillUIOrder": 0,
+      "properties": {
+        "database-name": {
+          "description": "Fill in your MongoDB database name",
+          "instillUpstreamTypes": [
+            "reference",
+            "template",
+            "value"
+          ],
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUIOrder": 0,
+          "title": "Database Name",
+          "type": "string"
+        },
+        "collection-name": {
+          "description": "The name of the collection in the MongoDB database",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillShortDescription": "Collection Name",
+          "instillUIOrder": 1,
+          "instillUpstreamTypes": [
+            "reference",
+            "template",
+            "value"
+          ],
+          "title": "Collection Name",
+          "type": "string"
+        },
+        "criteria": {
+          "description": "The criteria to find documents, if JSON value of a key is null e.g {'name':null}, then name field will be selected without any criteria, if JSON is empty e.g {}, then all fields will be selected",
+          "instillAcceptFormats": [
+            "semi-structured/*","structured/*","object"
+          ],
+          "instillShortDescription": "JSON Data",
+          "instillUIOrder": 2,
+          "instillUpstreamTypes": [
+            "reference",
+            "template",
+            "value"
+          ],
+          "title": "Criteria"
+        },
+        "limit":{
+          "description": "The number of documents to return, if 0 then all documents will be returned, otherwise the number of documents to return",
```

Comment:
Question

> if 0 then all documents will be returned, otherwise the number of documents to return",

How about we just make it optional & judge it in our codebase rather than declaring it in document?

0 turns out to be all seems not intuitive.
Null turns out to be all make more sense in my opinion.